### PR TITLE
Responsive web design: Milestone 2

### PR DESCRIPTION
**Pull request by Emmyn5600**
##### Emmy N.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Well done team! So far so good! :+1: There are just a few details to work on to complete this milestone. These include:
- [ ] Linting: I can't find the correct linter cofig files in your repository. Please make sure that you use the correct [linter config files](https://github.com/microverseinc/linters-config/tree/master/html-css) ([Link](https://microverse.zendesk.com/hc/en-us/articles/360055660994-Linters) to zendesk notice)
#### Latest news section
_Mobile:_
- [ ] For each article, the content on the right having the title above the author & time should be aligned to start at the same top level of the image. Your version appears to be positioned in the center. __Hint:__ Do exactly what you did for the `Latest Deals section`.

#### Categories (Apps, Gear, Tech, etc) section
_All screen sizes:_
- [ ] The title of each category should have a header tag (`hx`) and a link (`a` tag). Your current version made use of the `div` tag rather than the more semantically correct `hx` tag.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+- Change the current working directory to the location where you want the cloned directory to be made. Leave as it is if the current location is where you want the project to be.
+
+- Type git clone, and then paste the URL you copied in Step 3.
+$ git clone https://github.com/
+
+- Press Enter key
+
+- Press Enter. Your local copy will be created.
+
+Please Note that you must have github installed on your PC, this can be done here.
```
Comment (enaburekhan) [inline README.md]: I believe you meant git, and not github. Kindly correct this

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Changes required:full_moon:
**Hi team!
Great job! The milestone looks exactly like the original and UI is awesome:clap::clap:. Keep up the good job:thumbsup:. However, we still need to a make few changes to get the project approved:muscle:.**

**Mandatory requirements**

- [ ] Latest news section

**Mobile/Tablet**

- Each article has the image to the left, and the content on the right having the title above the author & time. As already highlighted by the previous Tse, The project version does not currently align properly to the left. It should align with the image. see [screen-shot](https://imgur.com/a/O7SJy5L) for reference

- [ ] Categories (Apps, Gear, Tech, etc) section

**All screen sizes**
- The title of each category has a header tag (`hx`) and a link (`a` tag). Wrap each of the titles in this category in a header.

- [ ] Check my comment below on the PR for a minor correction on the readme.

Make the above changes and when you are ready, request another review.
Kindly drop your questions :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hello Team :wave:
Status: Changes Required :recycle:

Good job. I like your design, I can see that you worked on the previous tse requirement. Your repo looks professional
But before approving this project let's cover a few details together to make your project better :blush: :arrow_double_down: .

REQUIRED:

- [ ] Fix linter error:


- [ ] Images in Lates funding, Hard Fork, Apps, Gear... and Latest deals are not displayed correctly:



That's it, you can submit another code review request after making these changes
With all my respect :pray:

Rindra Josia

**Comment by Emmyn5600** [comment]
Hey Josia,

It seems like this happened because of some issues with this clone website (
https://perma.cc/M5ZV-Q2D6), check that link it is not showing us the
images as well, and I used the link as the images.

I look forward to hearing from you,

Best,
Emmy N.

**Comment by Emmyn5600** [comment]
That is the cloned website.

**Review by bini-i — APPROVED** [review]
# Status: Approved :heavy_check_mark: 
### Hello  team :wave: 

#### Thanks for working on the requirements suggested by the previous TSE, and your project is approved.

Suggestions: 
-  you have linter errors as shown by the previous TSE, please work on them before merging this branch.
    <strong>hint</strong>: add \<meta charset="utf-8"> to the beginning of your head tag
             and always use header tags(h2, h3 ...) inside your section tags

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Pull request merged**